cmd/prometheus/main.go

```diff
@@ -656,6 +656,7 @@ func main() {
 	}
 	{
 		// TSDB.
+		opts := cfg.tsdb.ToTSDBOptions()
```

Comment:
It feels weird to me that we're delegating flag parsing for the tsdb in main to the web handler.

Reply:
I agree, happy to move it to main

Reply:
done.